pkg/utils/release/controller.go

```diff
@@ -39,50 +41,51 @@ func (s *SuiteController) CreateApplicationSnapshot(name string, namespace strin
 }
 
 // CreateRelease creates a new Release using the given parameters.
-func (s *SuiteController) CreateRelease(name, namespace, snapshot, sourceReleaseLink string) (*v1alpha1.Release, error) {
-	release := &v1alpha1.Release{
+func (s *SuiteController) CreateRelease(name, namespace, snapshot, sourceReleasePlan string) (*appstudiov1alpha1.Release, error) {
+	release := &appstudiov1alpha1.Release{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      name,
 			Namespace: namespace,
 		},
-		Spec: v1alpha1.ReleaseSpec{
+		Spec: appstudiov1alpha1.ReleaseSpec{
 			ApplicationSnapshot: snapshot,
-			ReleaseLink:         sourceReleaseLink,
+			ReleasePlan:         sourceReleasePlan,
 		},
 	}
 
 	return release, s.KubeRest().Create(context.TODO(), release)
 }
 
-// CreateReleaseLink creates a new ReleaseLink using the given parameters.
-func (s *SuiteController) CreateReleaseLink(name, namespace, application, targetNamespace, releaseStrategy string) (*v1alpha1.ReleaseLink, error) {
-	releaseLink := &v1alpha1.ReleaseLink{
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      name,
-			Namespace: namespace,
-		},
-		Spec: v1alpha1.ReleaseLinkSpec{
-			DisplayName:     name,
-			Application:     application,
-			Target:          targetNamespace,
-			ReleaseStrategy: releaseStrategy,
-		},
-	}
+// GetRelease returns the release with the given name in the given namespace.
+func (s *SuiteController) GetRelease(releaseName, namespace string) (*appstudiov1alpha1.Release, error) {
+	release := &appstudiov1alpha1.Release{}
 
-	return releaseLink, s.KubeRest().Create(context.TODO(), releaseLink)
+	err := s.KubeRest().Get(context.TODO(), types.NamespacedName{
+		Name:      releaseName,
+		Namespace: namespace,
+	}, release)
+
+	return release, err
 }
 
 // CreateReleaseStrategy creates a new ReleaseStrategy using the given parameters.
-func (s *SuiteController) CreateReleaseStrategy(name, namespace, pipelineName, bundle string, policy string) (*v1alpha1.ReleaseStrategy, error) {
-	releaseStrategy := &v1alpha1.ReleaseStrategy{
+func (s *SuiteController) CreateReleaseStrategy(name, namespace, pipelineName, bundle, policy, extraConfigGitUrl, extraConfigPath, extraConfigRevision, service_account string) (*appstudiov1alpha1.ReleaseStrategy, error) {
+	releaseStrategy := &appstudiov1alpha1.ReleaseStrategy{
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      name,
 			Namespace: namespace,
 		},
-		Spec: v1alpha1.ReleaseStrategySpec{
+		Spec: appstudiov1alpha1.ReleaseStrategySpec{
 			Pipeline: pipelineName,
 			Bundle:   bundle,
 			Policy:   policy,
+			Params: []appstudiov1alpha1.Params{
+				{Name: "extraConfigGitUrl", Value: extraConfigGitUrl, Values: []string{}},
+				{Name: "extraConfigPath", Value: extraConfigPath, Values: []string{}},
+				{Name: "extraConfigRevision", Value: extraConfigRevision, Values: []string{}},
+			},
+			// PersistentVolumeClaim: "test-pvc",
```

Comment:
what is this line for?

Reply:
It is part of the M7 demo files but I don't use it, so I'll delete the line

Reply:
Fixed